packages/orbit-components/src/List/ListItem/index.tsx

```diff
@@ -1,143 +1,25 @@
 import * as React from "react";
-import styled, { css } from "styled-components";
 
-import type { Type } from "../types";
-import type * as Common from "../../common/types";
-import type { Theme } from "../../defaultTheme";
-import defaultTheme from "../../defaultTheme";
 import CircleSmall from "../../icons/CircleSmall";
-import { rtlSpacing } from "../../utils/rtl";
-import { StyledCarrierLogo } from "../../CarrierLogo";
-import { SIZES, TYPES } from "../consts";
-import { StyledText } from "../../Text";
-import ListContext from "../ListContext";
 import type { Props } from "./types";
 
-export const getLineHeightToken = ({
-  theme,
-  size,
-}: {
-  theme: Theme;
-  size?: Common.Size | null;
-}): string | null => {
-  const lineHeightTokens = {
-    [SIZES.SMALL]: theme.orbit.lineHeightTextSmall,
-    [SIZES.NORMAL]: theme.orbit.lineHeightTextNormal,
-    [SIZES.LARGE]: theme.orbit.lineHeightTextLarge,
-  };
-  if (!size) return null;
-  return lineHeightTokens[size];
-};
-
-const getSizeTokenLabel = ({ theme, size }) => {
-  const sizeTokens = {
-    [SIZES.SMALL]: theme.orbit.fontSizeTextSmall,
-    [SIZES.NORMAL]: theme.orbit.fontSizeTextSmall,
-    [SIZES.LARGE]: theme.orbit.fontSizeTextNormal,
-  };
-  return sizeTokens[size];
-};
-
-const getIconSizeFromType = ({
-  theme,
-  type,
-}: {
-  theme: Theme;
-  type?: Type | null;
-}): string | null => {
-  const tokens = {
-    [TYPES.PRIMARY]: css`
-      height: ${theme.orbit.heightIconSmall};
-      width: ${theme.orbit.widthIconSmall};
-    `,
-    [TYPES.SECONDARY]: css`
-      height: ${theme.orbit.heightIconSmall};
-      width: ${theme.orbit.widthIconSmall};
-    `,
-  };
-
-  if (!type) return null;
-  return tokens[type];
-};
-
-export const Item = styled(({ ...props }) => <li {...props} />)`
```

Comment:
Unfortunately for now you would have to keep it, as it's required in other our components :) we will remove it once they also will be migrated ;)

Reply:
I solved it using `.orbit-list-item` in those other components